src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -1130,8 +1167,10 @@ emit_sys_numerics_vector (MonoCompile *cfg, MonoMethod *cmethod, MonoMethodSigna
 	MonoType *etype;
 
 	id = lookup_intrins (vector_methods, sizeof (vector_methods), cmethod);
-	if (id == -1)
+	if (id == -1) {
```

Comment:
There are same checks in `emit_sri_vector`, `emit_vector64_vector128_t` and `emit_vector256_t`. Could you add it to those functions too?